logspout_container_definition.json

```diff
@@ -0,0 +1,14 @@
+[
+  {
+    "name": "${task_name}",
+    "image": "gliderlabs/logspout:latest",
```

Comment:
The last time I used logspout their upgrade from v2 to v3 broke things for us because we used `:latest`. What do you think about locking this to the current image version we've tested it with? Looks like that would be `:v3.2.4` in this case: https://hub.docker.com/r/gliderlabs/logspout/tags/